### PR TITLE
support build dbpack locally && use alpine as base image

### PR DESCRIPTION
**Pull request by zackzhangkai**
support build dbpack locally && use alpine as base image

Signed-off-by: zackzhangkai <zhangkaiamm@gmail.com>

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -65,28 +65,27 @@ unit-test: ## run unit test
...
+	${GO_BUILD_ENVVARS} go build -o dbpack ./cmd
 
 ########################################################
-build:  ## build dbpack cli, and put in dist dir
-	@mkdir -p dist
-	GOOS="linux"  GOARCH="amd64" CGO_ENABLED=0 go build -o ./dist/dbpack ./cmd
+docker-build: ## build docker image
+	go mod download
+	go mod tidy
+	docker build -f docker/Dockerfile-rws -t dbpack:latest .
```
Comment (dk-lockdown) [inline makefile]: docker-build should use a common dockerfile

Comment (gorexlv) [reply makefile]: how about importing docker build-tags?

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+RUN make build-local
 
-COPY dist/ .
-
-CMD ["./dbpack", "start", "-c", "config.yaml"]
\ No newline at end of file
+FROM alpine:latest
+WORKDIR /
+COPY --from=builder /app/dbpack /dbpack
+COPY --from=builder /app/docker/scripts/wait-for-mysql.sh /wait-for-mysql.sh
```
Comment (dk-lockdown) [inline docker/Dockerfile]: wait-for-mysql should not copy to the dockerfile

**Review by dk-lockdown — APPROVED** [review]
LGTM

**Pull request merged**